Exercise_A/index.html

```diff
@@ -17,29 +18,75 @@
+                <a class="page-scroll" href="#portfolio">Bullshit</a>
+                <a class="page-scroll" href="#contact">Call Me</a>
+            </div>
+        </div>
+    </nav>
 
-    <!-- Can use Jumbotron -->
-    <!-- Second Section -->
+    <header>
+        <div class="header-content">
+            <div class="header-content-inner" id="content">
+                <h1 class="animate__animated animate__bounce animate__infinite">Hey Look, It's Every Bootstrap Website
+                    Ever</h1>
+                <p>Take a look around at the same fucking bootstrap page you've seen ten million times before! Featuring
+                    all the same things as those pages!</p>
+                <a href="#about" class="btn btn btn-warning page-scroll">This button makes the page scroll</a>
+            </div>
+        </div>
+    </header>
 
-    <!-- Four icons -->
-    <!-- Four pictures with overlay etc -->
-    <!-- contact me container -->
+    <section class="bg-primary" id="about">
+        <div class="container">
+            <div class="row">
+                <div class="col-lg-8 col-lg-offset-2 text-center">
+                    <h2 class="section-heading">Want to make an original website yourself?</h2>
+                    <p class="text-faded">Forget that! Who would ever want to put in all of that effort for a website?
+                        Just open up your web browser and type "bootstrap template" into your favorite search engine,
+                        like Yahoo! or Bing, and you're on your way! There are hundreds of templates to choose from, but
+                        go ahead and pick this same exact template from the first result on google, edit a few lines,
+                        and you're on your way! No one will notice!</p>
+                    <a href="http://lmgtfy.com/?q=bootstrap+template" target="_blank" class="btn btn-warning"
+                        id="google">Google
+                        That Shit</a>
+                </div>
+            </div>
+        </div>
+    </section>
 
-    <!-- All scripts we need jquery, bootstrap, smooth scroll, local script -->
-
-    <!-- 
-      All of your script tags for your JavaScript Bootstrap components, refer to the Practice folder
+    <!-- Bootstrap Core JavaScript -->
+    <script src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.10.2/dist/umd/popper.min.js"
+        integrity="sha384-7+zCNj/IqJ95wo16oMtfsKbZ9ccEh31eOz1HGyDuCQ6wgnyJNSYdrPa03rtR1zdB"
+        crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/js/bootstrap.min.js"
+        integrity="sha384-QJHtvGhmr9XOIpI6YVutG+2QOK9T+ZnN4kzFN1RtK3zEFEIsxhlmWl5/YESvpZ13"
+        crossorigin="anonymous"></script>
+    <!-- WOW JS -->
+    <script src="/Exercise_A/assets/js/wow.js"></script>
+    <script>new WOW().init();</script>
+    <!-- Custom Theme JavaScript -->
+    <script src="/Exercise_A/assets/js/script.js"></script>
+</body>

```

Comment:
Good job on using the animation for the website, though I would like to add that the middle section of your page is not aligned in the middle. For this, you can use Flex to align it in the Center. Also, the bottom parts are not complete, where you would need to add images and contact information.